### ISSUE TITLE
Writing file targets using here()

### ISSUE THREAD

**Issue by tiernanmartin** [ropensci/drake#249]
[`here::here()`](https://github.com/krlmlr/here/) is really useful for standardizing filepath roots within a project, but I'm not sure the best way to incorporate it into my drake plan.

Here's an example that works, albeit with a warning:

``` r
library(drake)
library(here)
library(tidyverse) 

here()
## [1] "file/path/from/root/directory" <- Note: this is output will be different for every system

drake_here <- function(x) {
  x %>% str_replace_all("'", "") %>% here() %>% as_drake_filename()
}

plan <- drake_plan(
  'tmp/mtcars.csv' = write_csv(mtcars, here("tmp/mtcars.csv")),
  file_targets = TRUE,
  strings_in_dots = "literals"
) %>% purrr::modify_at("target", drake_here)

make(plan)

```

Here's the warning I'm seeing:

``` r
## Warning message:
## Possibly bad target names:
##  file/path/from/root/directory/tmp/mtcars.csv   
```

Does this approach seem like a good one or would you recommend something else?

**Comment by wlandau** [comment]
I think it's a great approach, and it will become easier when #232 is fixed. The warning is my fault. I make a point of checking that non-file target names should be symbols you could use as variables in R, but the same need not apply to files. I will roll in a fix with #250.

**Comment by tiernanmartin** [comment]
#232 broke `drake_here()` as defined in my previous post.

The revised version below works:

``` r

library(drake)
library(here) 
library(tidyverse) 

drake_here <- function(x) {
  x %>% str_replace_all('\"', "") %>% here() %>% as_drake_filename()
}

plan <- drake_plan(
  write_csv(mtcars, file_out(here("mtcars.csv")))
) %>% modify_at("target", drake_here)

make(plan)
## cache C:\Users\UrbanDesigner\AppData\Local\Temp\RtmpqqMu01\.drake
## connect 2 imports: drake_here, plan
## connect 1 target: file "mtcars.csv"
## check 2 items: mtcars, write_csv
## check 1 item: file "mtcars.csv"
## target file "mtcars.csv"
```
